python/python/bystro/ancestry/upload_model.py

```diff
@@ -0,0 +1,49 @@
+"""Upload PCA and RFC models to ancestry s3 bucket."""
```

Comment:
This script is basically pure IO so it's not a great candidate for unit testing (but change my view?).  The script works and the model products are already uploaded-- I'm just checking it in for posterity / automating future deployments.